workflows/grzqc.nf

```diff
@@ -39,104 +36,6 @@ workflow GRZQC {
     ch_versions = Channel.empty()
     ch_multiqc_files = Channel.empty()
 
-    //
-    // conflicting parameters
-    //
-    // Error if BWA or FAI is given without FASTA and reference_path is not present
-    if ((params.bwa || params.fai) && !params.fasta && !params.reference_path) {
```

Comment:
do we have a check if the references are complete when we use reference_path?

Reply:
so, this implementation, only checks if reference_path is there or not? and if not it creates everything from input fasta?

Reply:
Currently, it only creates things that don't exist in the reference_path (if a reference_path is given). So if FASTA is deleted but bwamem2 is not then FASTA will be re-bgziped but bwamem2 will not be recreated. I'm open to forcing everything to be regenerated if the FASTA is missing but it would increase complexity. Though it would also prevent many potential bugs